src/class-wpml-vc-website-builder-translatable-nodes.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * Class WPML_VC_Website_Builder_Translatable_Nodes
+ */
+class WPML_VC_Website_Builder_Translatable_Nodes implements IWPML_Page_Builders_Translatable_Nodes {
```

Comment:
@adrianosferreira This file in nearly identical to `WPML_Beaver_Builder_Translatable_Nodes`. I see that $settings is an array here and it's an object in BB. Can't we reuse code somehow for this?

The only real difference is the `initialize_nodes_to_translate` function

Reply:
@brucepearson basically the only difference is this one you said, object and array. I will look into it.